themes/vscode/vscode.theme.sh

```diff
@@ -26,4 +26,6 @@ _omb_theme_vscode_initialize() {
     unset -f _omb_theme_vscode_initialize
 }
 _omb_theme_vscode_initialize
+
+_omb_theme_PROMPT_COMMAND() { true; }
```

Comment:
This change is related to the internals of OMB, but we need to clear `_omb_theme_PROMPT_COMMAND` set by another theme in case that the theme has been switched from another to `vscode` in a single session.